packages/dds/tree/src/test/simple-tree/api/getSimpleSchema.spec.ts

```diff
@@ -24,12 +24,14 @@ const simpleString: SimpleLeafNodeSchema = {
 	leafKind: ValueSchema.String,
 	kind: NodeKind.Leaf,
 	metadata: {},
+	persistedMetadata: undefined,
 };
 
 const simpleNumber: SimpleLeafNodeSchema = {
 	leafKind: ValueSchema.Number,
 	kind: NodeKind.Leaf,
 	metadata: {},
+	persistedMetadata: undefined,
```

Comment:
Seeing that the transient metadata is always initialized to `{}` makes me wonder if we should do the same for the persisted one for consistency. Make it required instead of optional at the simple schema layer and default it to an empty object when mapping an undefined from the stored schema layer.